vendors/tpl-systemes/drivers/birdy-slim-iot/package-lock.json

```diff
@@ -0,0 +1,6319 @@
+{
```

Comment:
The package-lock.json file should be removed or regenerated from the new package.json instead of copying the one from template.